python/setup.py

```diff
@@ -7,8 +7,8 @@
 import os
 import shutil
 
-_major = '0.2'
```

Comment:
too late now, but shouldn't this be minor and patch? Major version right now is 0

Reply:
sorry, was just using the convention from the AzureML SDK.  Can update this to be major/minor/patch in the setup.py.